plugins/node/opentelemetry-instrumentation-redis/src/utils.ts

```diff
@@ -146,7 +147,10 @@ export const getTracedInternalSendCommand = (
         }
 
         endSpan(span, err);
-        return originalCallback.apply(this, arguments);
+        const callbackThis = this;
+        return context.with(originalContext, () => {
+          return originalCallback.apply(callbackThis, arguments);
+        });
```

Comment:
You can also just do `return context.with(originalContext, originalCallback, callbackThis, ...arguments)` https://open-telemetry.github.io/opentelemetry-js-api/classes/contextapi.html#with

Reply:
This way you avoid creating an additional function call

Reply:
Wasn't aware of that one. thanks @dyladan , fixed :)